src/app/ranking/ranking-tool/ranking-tool.component.ts

```diff
@@ -116,6 +125,15 @@ export class RankingToolComponent implements OnInit {
     }
   }
 
+  /**
+   * Update fixedPanel based on whether the scroll position is at the end of the element
+   */
+  private onScroll() {
+    const content = this.el.nativeElement.querySelector('.page-content');
+    this.fixedPanel = content.getBoundingClientRect().bottom >
```

Comment:
A note on performance here, we should probably calculate the position on the page where to fix the element in `ngAfterViewInit` and on window resize so it is not calculated on every call to `onScroll`.  Calls to `getBoundingClientRect()` are pretty costly, see here: http://dcousineau.com/blog/2013/09/03/high-performance-js-tip/